src/rpc/net.cpp

```diff
@@ -169,6 +170,7 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
             obj.pushKV("inflight", heights);
         }
         obj.pushKV("whitelisted", stats.fWhitelisted);
+        obj.pushKV("minfee", ValueFromAmount(stats.minFeeFilter));
```

Comment:
Missing test. Also, this feature should probably be submitted in separate pull request.